cmake/std/atdm/cee-rhel6/environment.sh

```diff
@@ -80,6 +72,9 @@ if [[ "$ATDM_CONFIG_COMPILER" == "CLANG-5.0.1_OPENMPI-1.10.2" ]]; then
   export ATDM_CONFIG_MKL_ROOT=${CBLAS_ROOT}
 elif [[ "$ATDM_CONFIG_COMPILER" == "GNU-7.2.0_OPENMPI-1.10.2" ]] ; then
   module load sparc-dev/gcc-7.2.0_openmpi-1.10.2
+  unset OMP_NUM_THREADS  # SPARC module sets these and we must unset!
```

Comment:
Yeah, if you do `ctest -j N` with `N > 1` and run with threads, binding threads to anything would be Quite Ungood.